cli/index.js

```diff
@@ -40,9 +42,33 @@ module.exports = function( config ) {
 
 	function getFileMatches( inputFiles ) {
 		return inputFiles.map( function( inputFile ) {
+			var inputComment;
+			var input = fs.readFileSync( inputFile, 'utf8' );
+
+			if ( ! config.ignoreFileComments ) {
+				var rxFileCommentMatch = new RegExp( "^//\\s((Screenshot|Demo) URL:\\s(.*))$", 'm' );
+				if ( rxFileCommentMatch.test( input ) ) {
+					var commentMatches = input.match(rxFileCommentMatch);
+					if ( validUrl.isWebUri( commentMatches[3] ) ) {
+						inputComment = commentMatches[1];
+					} else {
+						throw new Error( 'File comment must be a valid web uri' );
```

Comment:
Do we really want to throw here? Sounds like just skipping the comment silently would be a smoother solution

Reply:
👍 updated!